src/App.tsx

```diff
@@ -21,20 +21,36 @@ const App: React.FC = () => {
 
   const [inputValue, setInputValue] = React.useState<string>('')
 
- 
   const handleTask = () => {
-    
     setAllTask([...allTask, { title: inputValue, id: uuidv4(), checkedStatus: false }])
-    
   }
 
   const handleInput = (event) => {
     setInputValue(event.target.value)
   }
 
+  const updateSingleTask = (updatedTask: Task) => {
+    //go inside allTask array state variable
+    // find the element from state variabele allTask and use array.map for changing a single value in the array
+    // then use setAllTask from useState to update the state variable
+
+    // For example - {id: 12, title : 'hello', checkStatus: true} 
+    // updated Object {id:12 , title: 'hello updated , checkStatus: false}
+
+    allTask.map((task) => {
```

Comment:
store the updated array in a variable and pass that in the setAllTask on line 48